crates/tako/src/server/client.rs

```diff
@@ -199,15 +168,8 @@ pub async fn process_client_message(
                 .is_ok());
             None
         }
-        FromGatewayMessage::GetOverview(overview_request) => {
-            log::debug!("Client ask for overview");
-            let worker_overviews =
-                fetch_worker_overviews(core_ref, comm_ref, overview_request).await;
-            assert!(client_sender
-                .send(ToGatewayMessage::Overview(CollectedOverview {
-                    worker_overviews
-                }))
-                .is_ok());
+        FromGatewayMessage::GetOverview(_) => {
```

Comment:
You removed the implementation, so just also remove the {gateway, client} GetOverview RPC messages too.